tests/docs/private.pem

```diff
@@ -0,0 +1 @@
+
```

Comment:
Lets put this in gitignore for the external repo.

Reply:
Done